core/src/main/resources/hudson/util/HudsonIsRestarting/index_de.properties

```diff
@@ -1,5 +1,5 @@
 Please\ wait\ while\ Jenkins\ is\ restarting=Jenkins wird neu gestartet. Bitte warten.
```

Comment:
When changing the English text we should update / remove translations. 

e.g. here I think you would drop the `Bitte warten`

Reply:
I can confirm, it should be "Jenkins wird neu gestartet"

Reply:
```suggestion
Please\ wait\ while\ Jenkins\ is\ restarting=Jenkins wird neu gestartet.
```

Reply:
Should the "." be part of the text or not?